content-repo/extra-docs/articles/trust-custom-certificates-engines.md

```diff
@@ -0,0 +1,18 @@
+---
+id: trust-custom-certificates-engines
+title: Configure Engines to Trust Custom Certificates
+description: Replace the self-signed certificate for an engine with a valid CA certificate.
+---
+Engine initiated communication, Javascript integrations, and native integrations use the built-in set of CA-Signed certificates of the host machine to validate TLS communication.
```

Comment:
JavaScript, but not sure we should say that

Reply:
When customers do this task - replace the self-signed certificate for an engine with a valid CA certificate - what is the most likely reason?

Reply:
To secure the connection between the server to engine (to avoid man in the middle).

Reply:
https://en.wikipedia.org/wiki/Self-signed_certificate

Reply:
we usually try to add an example of a more specific use case, if applicable. if it's not needed here, I can change the beginning to just be:
To secure the connection between the server and the engine, you can replace the self-signed certificate for an engine with a valid CA certificate.
1. Find the two files created by the engine. The default location is /usr/local/demisto.  
...